src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,34 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int LENGTH_ARRAY = 10;
+    private int counter = 0;
+    private Object[] key = new Object[LENGTH_ARRAY];
+    private Object[] value = new Object[LENGTH_ARRAY];
```

Comment:
let’s initialise all fields in the constructor
```suggestion
    private int counter = 0;
    private Object[] keys = new Object[LENGTH_ARRAY];
    private Object[] values = new Object[LENGTH_ARRAY];
```

Reply:
done